flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -661,6 +673,18 @@ public boolean dropTemporaryView(String path) {
         }
     }
 
+    @Override
+    public boolean dropView(String path) {
+        UnresolvedIdentifier unresolvedIdentifier = getParser().parseIdentifier(path);
+        ObjectIdentifier identifier = catalogManager.qualifyIdentifier(unresolvedIdentifier);
+        try {
+            catalogManager.dropView(identifier, false);
+            return true;
+        } catch (ValidationException e) {
+            return false;
```

Comment:
I guess the before the fix, this exception would have bubbled up the table API. After the fix it is gobbled. I am thinking we should log the exception here, so we have an indication of what failed somewhere. WDYT?

Reply:
>I guess the before the fix, this exception would have bubbled up the table API.

can you elaborate what you mean by this? Before the PR this method wasn't exists so it could not bub up

Reply:
Yes you are right it is not a regression - still I think it is worth a log entry when there is an Exception and it returns false;